### ISSUE TITLE
[BUG] - Cardano-node crashes

### ISSUE THREAD

**Issue by os11k** [input-output-hk/cardano-node#4640]
**External**

**Area**
*Other*   Any other topic (Delegation, Ranking, ...).

**Summary**
Cardano-node 1.35.4 crashes with:

```
cardano-node: internal error: Itimer: Failed to spawn thread: Operation not permitted
	
Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug
	
(GHC version 8.10.7 for x86_64_unknown_linux)
```

**Steps to reproduce**
run cardano-node docker 1.35.4

**Expected behavior**
cardano-node starts normally

**System info (please complete the following information):**
official IOHK cardano node 1.35.4 docker image

**Additional context**
I have no problem to run that image on different physical server. Older images 1.35.3 has no such issue and works fine.

Master works too - cardano-node@sha256:725082fd97e3b71bb56342ef2941592c3c0b18f3667ef0147c5ed79bf957494e

Seems problem is just with 1.35.4 on one particular server...

**Comment by os11k** [comment]
Seems that bug appeared starting with 1.35.4-rc1

**Comment by Jimbo4350** [comment]
> I have no problem to run that image on different physical server. 

What are the specs?

**Comment by os11k** [comment]
I have problem it running on DH67BL Intel motherboard with Intel(R) Xeon(R) CPU E3-1245

**Comment by disassembler** [comment]
TLDR; upgrade docker

This issue was filed upstream to GHC and was reproducible using nix:
https://gitlab.haskell.org/ghc/ghc/-/issues/21604

It was tracked down to a bug in docker identified here: https://github.com/moby/moby/issues/42680

The issue occurs because dependencies were updated and those dependencies used a syscall that docker didn't white list. It's been resolved in newer docker versions.

**Comment by os11k** [comment]
Thank you @disassembler ! You are correct, newer version made it work. I was a bit confused that master worked, but not 1.35.4...
